src/6.7/6.7.2.0.md

```diff
@@ -0,0 +1,412 @@
+
+### Symfony profiler
+
+Complete cart data like line items, price calculations, transactions etc. can now be observed in Symfony profiler toolbar. Now, you can get a complete, clean, and organized view of the cart's state on any page load without interrupting the execution flow, dramatically speeding up development and troubleshooting.
+
+### Cart rule loader extension event
+
+The `RuleLoaderEvents::LOADER_EXTENSION_EVENT` is a powerful new hook that allows developers to bypass the limitations of the visual rule builder. It injects custom, PHP-based business logic directly into the process that determines which promotions are applied to the cart, greatly expanding the possibilities for creating sophisticated and dynamic promotions.
+
+This event is an "extension point." It doesn't let you change the rules themselves, but it allows you to add entirely new conditions or logic on top of the existing rule system.
+
+The classic problem it solves, for example, could be 
+
+* Discount applies if it's the user's birthday
+
+### Custom route names
+
+Yet another big win for developers, specifically those needing more control over URL generation and SEO in their Shopware themes and plugins. It allows developers to assign a unique, custom identifier to any page route in the storefront. This custom name can then be used to generate the URL for that page anywhere in the code, making it much more reliable, robust and maintainable.
+
+Now, anywhere in your code (Twig templates, controllers, services), you can generate the URL for this page using the clear, semantic name you chose (`frontend.deals.of.the.day`) instead of the less descriptive internal ID (`your_plugin.deals.page`).
+
+This change gives developers the freedom to define meaningful, project-specific names for their routes, leading to cleaner, more maintainable, and less brittle code when dealing with URLs in the storefront.
+
+### App custom fields
+
+This is a straightforward and powerful extension of the App API. It grants app developers two new "hooks" (unit and newsletter_recipient) to integrate more deeply with Shopware's data model, enabling more sophisticated and specialized applications.
+
+It extends that capability to two new, important entities:
+
+* unit: The entity that represents a product's measurement unit (e.g., "kg", "piece", "liter", "cm").
+* newsletter_recipient: The entity that stores information about someone who has signed up for a newsletter.
+
+App developers can now attach their own custom data to these entities.
+
+### Commands to schedule and deactivate scheduled tasks
+
+These new commands are powerful administrative tools. They move control of Shopware's background processes from a purely code-based configuration to something a system administrator or developer can manage dynamically via the command line, greatly improving flexibility for maintenance, debugging, and testing.
```

Comment:
honestly i'm wondering if we need all this AI fluff how great the changes are, they are super exagerating and do not add no information at all, can't we just document what is now possible and let the reader decide how useful he thinks that is?